frontend/src/app/shared/shared.module.ts

```diff
@@ -2,21 +2,30 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterModule } from '@angular/router';
+import { FaIconLibrary, FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { fab } from '@fortawesome/free-brands-svg-icons';
+import { far } from '@fortawesome/free-regular-svg-icons';
+import { fas } from '@fortawesome/free-solid-svg-icons';
```

Comment:
are you sure we need all of them?

Reply:
From brands I think we need github or google icons for login/signup page, and solid/regular we can remove one from them.

Reply:
Okay, lets try all of them and then we will decide which one we will remove